go/signature/verifier_factory.go

```diff
@@ -22,8 +22,8 @@ import (
 	"github.com/google/tink/go/core/primitiveset"
 	"github.com/google/tink/go/core/registry"
 	"github.com/google/tink/go/keyset"
-	"github.com/google/tink/go/tink"
 	tinkpb "github.com/google/tink/go/proto/tink_go_proto"
```

Comment:
could you also revert this change? it'll interfere with our merging process.

Reply:
weird, but this is done by normal `go fmt` .. I'll revert

Reply:
reverted change